src/main/kotlin/money/Money.kt

```diff
@@ -0,0 +1,9 @@
+package money
+
+import java.math.BigDecimal
+
+class Money(private val amount: BigDecimal) {
```

Comment:
행위의 결과로 상태 값이 존재하게 되는데, 아직 `Money`의 행위가 정의되지 않아 `amount`를 상태값으로 저장하지 않아도 될 듯 합니다. ㅎㅎ 

```suggestion
class Money(amount: BigDecimal) {
```